clients/tango-rest.py

```diff
@@ -185,17 +194,21 @@ def tango_addJob():
         requestObj['max_kb'] = args.maxsize
         requestObj['output_file'] = args.outputFile
         requestObj['jobName'] = args.jobname
+        
         if (args.notifyURL):
             requestObj['notifyURL'] = args.notifyURL
+        
+        requestObj['accessKeyId'] = args.accessKeyId
+        requestObj['accessKey'] = args.accessKey
 
+        print "Sent request to %s:%d/addJob/%s/%s/ \t jobObj=%s" % (args.server, args.port, args.key, args.courselab, json.dumps(requestObj))
```

Comment:
If the request fails then the log message may be something like this:
`Sent request to...`
`Failed to send request to...`
Both log messages will show the same details about what the log message is. Thus, even if the request fails we'll know what request it tried to make. The "Sent" and "Failed" is also slightly confusing because the "Sent" implies it succeeded.